chrome/content/zotero/preferences/preferences_proxies.js

```diff
@@ -56,6 +56,14 @@ Zotero_Preferences.Proxies = {
 	
 	
 	/**
+	 * Enables UI buttons when proxy is selected
+	 */
+	enableProxyButtons: function () {
+		document.getElementById('proxyTree-edit').disabled = undefined;
+		document.getElementById('proxyTree-delete').disabled = undefined;
```

Comment:
I don't think there's any functional difference, but just for convention, these should be `false`, not `undefined`. (You can just commit --amend that and force-push. This looks great otherwise.)